internal/hcsoci/create.go

```diff
@@ -145,6 +146,7 @@ func CreateContainer(ctx context.Context, createOptions *CreateOptions) (_ cow.C
 			if err != nil {
 				return nil, r, err
 			}
+			isSandbox = ct == oci.KubernetesContainerTypeSandbox
```

Comment:
Shouldn't this be moved out of the networking setup code path?

Reply:
Sure, now that it's used for more than just network setup

Reply:
Donezo